.gitignore

```diff
@@ -12,10 +12,3 @@ build/
 *.gcda
 *.gcno
 *.gcov
-
```

Comment:
These are deleted because tools/aws-templates-for-cbmc-proofs/scripts/setup.py will copy a .gitignore into the library's cbmc directory.